lib/Smartcat/App/Command/push.pm

```diff
@@ -96,13 +102,22 @@ sub execute {
                 utf8::decode($name); # assume UTF8 filenames
                 utf8::decode($_);
             }
+
             if (   -f $name
                 && !m/^\.$/
                 && m/$rundata->{filetype}$/ )
             {
                 s/$rundata->{filetype}$//;
                 my $path = catfile( dirname($name), $_ );
-                my $key = &get_ts_file_key($rundata->{project_workdir}, $path);
+
+                # print "\tPath: ".$path."\n";
+
+                my $key = $rundata->{split_filename_by_triple_dash}
+                    ? &get_ts_file_key($rundata->{project_workdir}, $path, "true" )
+                    : &get_ts_file_key($rundata->{project_workdir}, $path);
+
+                # print "\tKey: ".$key."\n";
```

Comment:
Here and below: let's remove debug printouts from the code.

Reply:
Debug printouts removed